libs/vkd3d/command.c

```diff
@@ -772,9 +772,6 @@ static HRESULT d3d12_fence_signal(struct d3d12_fence *fence, uint64_t physical_v
         return hresult_from_errno(rc);
     }
 
-    /* With multiple fence workers, it is possible that signal calls are
```

Comment:
Was it the intention to remove this comment here?

Reply:
Ah, no, must have screwed up while rebasing.